src/index.css

```diff
@@ -26,5 +26,5 @@ code {
 	align-items: center;
 	background-size: 1%;
 	background-image: url("data:image/svg+xml,%3Csvg width='20' height='20' viewBox='0 0 20 20' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='%23000000' fill-opacity='0.2' fill-rule='evenodd'%3E%3Ccircle cx='3' cy='3' r='3'/%3E%3Ccircle cx='13' cy='13' r='3'/%3E%3C/g%3E%3C/svg%3E");
-    cursor:grab;
+    
```

Comment:
I think you are on an old branch, please sync the fork
![image](https://user-images.githubusercontent.com/9199763/195389278-9caf1d3d-7172-4eaf-815d-675ca8d08cfd.png)

Reply:
It shows i have to delete 3 commits, do i delete it?

Reply:
I guess